src/openms/include/OpenMS/ANALYSIS/MAPMATCHING/TransformationModel.h

```diff
@@ -93,6 +93,7 @@ namespace OpenMS
     /// Alternative constructor (derived classes should implement this one!)
     /// Both data and params must be provided, since some derived classes require both to create a model!
     TransformationModel(const TransformationModel::DataPoints&, const Param&);
+    TransformationModel(const std::vector<std::pair<double,double>>& data, const Param&);
```

Comment:
why ?

Reply:
unused. I wanted to use it for interpolation of ROC curves I think and offer a second constructor without packing and right-away unpacking x and y values. Was too complex in the end.
Removed.